org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/tree/AbstractTreeNode.java

```diff
@@ -699,8 +621,6 @@ public List<IMenu> getMenus() {
 
   protected List<IMenu> lazyCreateAndInitializeMenus() {
     List<Class<? extends IMenu>> declaredMenus = getDeclaredMenus();
-    List<IMenu> contributedMenus = m_contributionHolder.getContributionsByClass(IMenu.class);
```

Comment:
Menu contributions do not work anymore on pages as well. I think this is not intended. Create a new `contributeMenusInteral` method that returns an empty list on `AbstractTreeNode`, but that uses the `contributionHolder` on `AbstractPage`.